src/js/common/schemaform/save-load-actions.js

```diff
@@ -99,12 +102,14 @@ export function migrateFormData(savedData, savedVersion, migrations) {
  * @param  {Object}  formData  The data the user has entered so far
  */
 export function saveInProgressForm(formId, version, returnUrl, formData) {
+  const savedAt = Date.now();
   // Double stringify because of api reasons. Olive Branch issues, methinks.
   // TODO: Stop double stringifying
   const body = JSON.stringify({
     metadata: JSON.stringify({
       version,
-      returnUrl
+      returnUrl,
+      savedAt
```

Comment:
@cvalarida I'm stealing what you did in your PR, since it makes more sense for me to use this date than get something from the http response.